languages.toml

```diff
@@ -1038,3 +1038,21 @@ roots = []
 comment-token = "//"
 indent = { tab-width = 4, unit = "    " }
 grammar = "rust"
+
+[[language]]
+name = "gdscript"
+scope = "source.gdscript"
+injection-regex = "gdscript"
+file-types = ["gd"]
+shebangs = []
+roots = ["project.godot"]
+auto-format = true
+comment-token = "#"
+indent = { tab-width = 4, unit = "    " }
+grammar = "gdscript"
+# TODO: change this when socket support is added
+language-server = { command = "nc", args = ["localhost", "6008"] }
```

Comment:
This is an interesting hack. I'm a little worried it won't be cross-platform: does windows have netcat?

I think until socket support is added, it makes sense to remove the `language-server` configuration here and configure it in your `~/.config/helix/languages.toml`, maybe with some notes in the new [lsp wiki](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers). What do you think?

Reply:
That sounds good.
Maybe I could write a python hack that's a bit more complicated but cross-platform.
Or a binary could be included with helix for this.
But until then, I agree, this should be removed.